src/components/Contract-Status-Widget.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useQuery, gql } from "@apollo/client";
+import { Box, Typography } from "@mui/material";
+
+const GET_CONTRACT_STATUS_COUNTS = gql`
+  query GetContractStatusCounts {
+    contractStatusCounts {
+      notStarted
+      inProgress
+      done
+    }
+  }
+`;
```

Comment:
This needs to be in a seperate file - see how `src/pages/SignupPage/signup.js` is done

Reply:
turned into separate file